### ISSUE TITLE
Cannot open load file: no such file or directory, generator

### ISSUE THREAD

**Issue by tuhdo** [michael-heerdegen/helm-browse#1]
It seems like this module is missing and is not part of Emacs.

**Comment by michael-heerdegen** [comment]
Thanks for volunteering trying this...

 generator.el has been added to Emacs 25 recently, so you need a recent built of Emacs trunk.  Changes to the C code of Emacs were part of the addition of generators, so helm-browse will only work with Emacs 25, sorry.  Probably, when this package will have been finished, Emacs 25 will have been released (hopefully).

Do you have access to an Emacs 25 installation?  In any case, thanks for trying.

**Comment by tuhdo** [comment]
Yes, I do. If this can be used in Emacs 25, it will be difficult to integrate within Helm.

**Comment by michael-heerdegen** [comment]
Mhm, right.  OTOH there is no hurry to integrate it into helm.

I could rewrite the code to work without generator.el and iterators.el, but this would lose an abstraction layer and make the code look a bit uglier.

**Comment by michael-heerdegen** [comment]
I updated README to say that Emacs 25 is required.
